content/ko/examples/service/networking/example-ingress.yaml

```diff
@@ -2,8 +2,6 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: example-ingress
-  annotations:
-    nginx.ingress.kubernetes.io/rewrite-target: /$1
 spec:
   rules:
```

Comment:
To make this example file fully synchronized with the ‘en’ [version](https://github.com/kubernetes/website/blob/main/content/en/examples/service/networking/example-ingress.yaml), we should also consider adding the missing `ingressClassName: nginx` field. This will ensure consistency and completeness.

Refer [en/examples/example-ingress.yaml#L6](https://github.com/kubernetes/website/blob/main/content/en/examples/service/networking/example-ingress.yaml#L6)

Reply:
Thanks, but that's in another PR https://github.com/kubernetes/website/pull/47159